dev/script/es-ssh-ssm-tunnel.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+function HELP {
+>&2 cat << EOF
+
+  Usage: ${0} -t ES-HOST [-s TEST] [-u ubuntu]
+
+  This script sets up an ssh tunnel from localhost port 9300 to the ElasticSearch provided on port 9300
```

Comment:
Why 9300 over 9200?

Reply:
I think it was in the README or something equally not good as an answer. Will fix.